lib/foreman/gettext/support.rb

```diff
@@ -3,6 +3,8 @@
 module Foreman
   module Gettext
     module Support
+      @@mutex = Mutex.new
+
```

Comment:
There also should be initialization of `@@locales` otherwise it crashes.
```suggestion
      @@locales = nil
```